### ISSUE TITLE
Raid array setup/detail

### ISSUE THREAD

**Issue by superedo73** [facebook/Surround360#45]
Hi,

I read the documentation but I miss the point about RAID setup.
You say 8 x 1TB SSD sata III HD in a RAID5 array.
Is there a detailed setup document about this configuration / may you better specify?

Anyway, why RAID5? 
Despite the proper redundancy, as far I know, write performance are poor compared to RAID0 (without redundancy) or RAID10 (at expense of much more disk used) - I've seen RAID5 often used in read-oriented arrays, while in FBS360 write performance seems the crucial point.

Thank you in advance for your reply.

Enrico

**Comment by superdog8** [comment]
Sounds like a mistake in the manual. RAID5 is indeed best for backup scenarios, while for capture, we use RAID0.

**Comment by superdog8** [comment]
superedo73, which documentation manual are you refering to ?

**Comment by superedo73** [comment]
You're right, there is no explicit indication about raid level in the doc.manual. Sorry, my mistake!

Anyway, I get the information here:
https://code.facebook.com/posts/1755691291326688/introducing-facebook-surround-360-an-open-high-quality-3d-360-video-capture-system
_"...Similarly, we used an 8-way level-5 RAID SSD disk system to keep up with the isochronous camera capture rates..."_  so I intended that RAID 5 has been chosen for capture operations.

My suggestion is to specify in the manual the suggested/best RAID setup.

Thank you very much!

Enrico
